Makefile

```diff
@@ -257,23 +257,23 @@ bounce-argo-pods:
 ## Secrets ##
 #############
 .PHONY: secrets-download
-secrets-download: pre-check
```

Comment:
In the context of secrets this precheck only has usefulness (IMO) to check that ENVIRONMENT is set and that is handled by the secrets.sh script.